corgi/core/models.py

```diff
@@ -421,17 +423,49 @@ def builds(self) -> QuerySet:
 
     @property
     def cpes(self) -> tuple[str, ...]:
-        """Return CPEs for all descendant variants."""
-        variant_cpes = (
+        """Return CPEs for direct descendant variants if they exist. Otherwise, return a union
+        of indirect descendant variants and descendant streams cpes_matching_patterns"""
+        if self is ProductVariant:
+            return (self.cpe,)  # type: ignore[attr-defined]
```

Comment:
I guess this check is superfluous because we override the cpes property for ProductVariant model. Although I think it doesn't hurt to keep it here. 
I'll remove the  include_self=True in the call to get_descendants() as it is redundant.